--- git status ---
On branch main
Your branch is up to date with 'origin/main'.

Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git restore <file>..." to discard changes in working directory)
	modified:   scripts/lynx/readme.sh

no changes added to commit (use "git add" and/or "git commit -a") 


--- git diff ---
diff --git a/scripts/lynx/readme.sh b/scripts/lynx/readme.sh
index 100e9a31..cc1ff539 100644
--- a/scripts/lynx/readme.sh
+++ b/scripts/lynx/readme.sh
@@ -13,6 +13,7 @@
 # Distributed training via rsl_rl:
 python -m torch.distributed.run --nnodes=1 --nproc_per_node=2 scripts/reinforcement_learning/rsl_rl/train.py --task=Isaac-Reach-Lynx-v0 --distributed --headless --num_envs 16384
 python -m torch.distributed.run --nnodes=1 --nproc_per_node=2 scripts/reinforcement_learning/rsl_rl/train.py --task=Isaac-Reach-Lynx-v2 --distributed --headless --num_envs 16384
+python -m torch.distributed.run --nnodes=1 --nproc_per_node=2 scripts/reinforcement_learning/rsl_rl/train.py --task=Isaac-Open-Drawer-Franka-v0 --distributed --headless --num_envs 16384
 
 # also works for 4070 ti super (16GB VRAM), the observation excludes joint velocities:
 python scripts/reinforcement_learning/rsl_rl/train.py --task=Isaac-Reach-Lynx-v1 --distributed --headless --num_envs 16384